rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -20,7 +20,28 @@ use hyperlane_core::HyperlaneDomain;
 
 use super::pending_operation::*;
 
-type OpQueue = Arc<Mutex<BinaryHeap<Reverse<Box<DynPendingOperation>>>>>;
+#[derive(Debug, Clone, new)]
```

Comment:
converted this alias to a newtype struct, since `push` occurs in several places and that would duplicate calls to metrics